### DISCUSSION TITLE
Bringing a Democratic Process To Node Management

### DISCUSSION THREAD

**Discussion by jam2ja** [deso-protocol/dips#126] [Medium CIPs]
Tonight in a club house room, ideas around Democracy within the bitclout network were discussed.

As I understand things, the person (group of people) that creates the node in the context of this bitclout project, controls how the node is managed, maintained and updated. The NFT nodes have a lot of users, yet those users/creators have no final say in the architecture of that node. Could this change?

With todays technology it's easy to bring ideas such as Direct Democracy, and liquid Democracy to this network.
https://en.wikipedia.org/wiki/Liquid_democracy

Why not?

-Users want to be involved in the decision making process's (changes in architecture) that effect the value of their input.
-The community is empowered when they are given the opportunity to make decision.

**Comment by tijno** [comment]
I think this is a decision for individual node operators and not something for the opensource code to force on all nodes.

**Reply by jam2ja** [reply]
If someone was to build an NFT node that did have governance, it can compete for attention against other nodes though. Governance and transparency seem to be what users are looking for?

 Yet i get that if you have made a node, and it's yours, you'd not wish to make a governance protocol for the users you have, it makes no sense. Yet in the end, this might be the reason that users goto other platforms..

**Reply by tijno** [reply]
i agree - but is this something that requires a change to bitclout frontend or the blockchain ? 

and if not - what is purpose of the cip? 

eg what is the change to bitclout you are proposing.

**Reply by marnimelrose** [reply]
> I think this is a decision for individual node operators and not something for the opensource code to force on all nodes.
> 

I am in agreement with this.

**Reply by jam2ja** [reply]
> and if not - what is purpose of the cip?

Maybe explore other networks and discussions on the issues. 
https://www.henvote.xyz/about
https://www.reddit.com/r/cardano/comments/m44hx5/cardano_self_sustainable_governance_and_liquid/

One suggestion might be that people who register with their name on the network, would get a vote. One member one vote, changes that effect users, all member votes. I like this idea of liquid democracy that Cardano is using though. I guess the system as it is, is not built for a democratic process, so i see what your saying.. 
> and if not - what is purpose of the cip? !!!!

**Reply by jam2ja** [reply]
> I think this is a decision for individual node operators and not something for the opensource code to force on all nodes.

the best situation might be to just add the idea of liquid and or direct democracy as a feature that can be turned off or on...within each node. or we want to keep this beast so top heavy)

**Comment by theparkerazzi** [comment]
@jam2ja I have CloutingAround.dev I would like to speak with you about this some more. Let's try it out node?

**Reply by jam2ja** [reply]
For sure its a good idea to introduce some sort of governance into the node your building, yet i'm not a coder. Building governance into nodes, seems like a good structure to have. Most of the proof of stake systems have made interesting governance projects.

**Comment by theparkerazzi** [comment]
It's proof. I agree Marni. working on it
